components/04-page-layouts/_base.twig

```diff
@@ -0,0 +1,32 @@
+{% set layout__base_class = 'layout-container' %}
+{% set content__base_class = 'main' %}
+{% set sidebar__base_class = 'sidebar' %}
+{% set main__base_class = 'main-content' %}
+
+<div {{ bem(layout__base_class) }}>
+  {# Page Header #}
```

Comment:
For the Header and Footer: can we use <header> and <footer> elements? These will provide more semantic meaning for screen reader users.

Reply:
Yes, we will. I haven't actually created the header and footer components yet (those are coming next sprint.) The semantic markup you mentioned will be in those components.

This "layout" component is using Twig "block"s which are basically "buckets" where components can be inserted.

All of the `<section class="placeholder">` elements will be entirely replaced.

Reply:
I started this PR this morning. Maybe it'll help understand things, in case I was just confusing up above. https://github.com/yalesites-org/component-library-twig/pull/18